setup.py

```diff
@@ -5,21 +5,21 @@
 
 setuptools.setup(
     name="reposit",
-    version="0.4.0",
+    version="0.5.0",
     author="Thomas Basche",
-    author_email="thomas.basche@repositpower.com",
+    author_email="tcbasche@gmail.com",
     description="A library to communicate with a Reposit Controller",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/RepositPower/reposit-python-client",
     packages=setuptools.find_packages(),
     install_requires=[
         'requests',
-        'pendulum',
         'six'
     ],
     classifiers=(
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
If also bumping the known-good and supported Python version to the just released Python 3.8 with this PR, it would also be worthwhile moving Python 3.8 to the must-pass list within `.travis.yml`.

(e.g. add `3.8` to the list of tested versions under `python:` in  `.travis.yml` and change the `allow_failures:` to upcoming `3.9-dev`)

I'd expect Travis-CI will update their environments in the next couple of days for the final Python 3.8.0 release.

Reply:
True! Should probably also include @switchtrue in this :)